doc/guides/using-symbols.md

```diff
@@ -0,0 +1,61 @@
+# Using global symbols
+
+ES6 introduced a new type: `Symbol`. This new type is _immutable_, and
+it is often used for metaprogramming purposes, as they can be used as
+property keys like string. There are two types of
+symbols, local and globals. Symbols are not included in
+JSON.stringify() but thay are by `util.inspect()`.
+
+## Local `Symbol`
```

Comment:
There is no such thing as a "local" Symbol. Just say symbols created using the `Symbol()` function.